autodepgraph/tests/test_graph.py

```diff
@@ -53,6 +53,13 @@ def test_maintain_node_assume_unkown_is_good(self):
         self.assertEqual(self.test_graph.nodes()['C']['state'], 'good')
         self.assertEqual(self.test_graph.nodes()['B']['state'], 'unknown')
 
+    def test_calibration_state(self):
+        self.test_graph.calibration_state()
+        
+    def test_set_function(self):
+        self.test_graph.set_node_attribute('A', 'myattribute', 10)
+        self.test_graph.set_node_description('A', 'explain node A')
```

Comment:
Maybe make the test read back that these are actually set?

Reply:
Done (and added `get_node_attribute`)